Geometry/TrackerCommonData/data/Materials/mixture.py

```diff
@@ -0,0 +1,385 @@
+#!/bin/python3
```

Comment:
@vargasa why not the usual #!/usr/bin/env python3 , provided we want to force python3?
@davidlange6 FYI, as we are in the middle of the migration